src/Particle/DistanceTableData.h

```diff
@@ -17,12 +17,13 @@
 #define QMCPLUSPLUS_DISTANCETABLEDATAIMPL_H
 
 #include "Particle/ParticleSet.h"
+#include <limits>
+#include <bitset>
```

Comment:
I think since you went the enum route bitset isn't needed.

Reply:
removed